File: win32/xampp/contrib/postgresql.php
<html>
<head>
<title>PHP und PostgreSQL</title>
</head>
<body>
<h1>PHP und PostgreSQL</h1>
<table border="1">
<tr>
    <th>Interpret</th>
    <th>Titel</th>
    <th>Jahr</th>
</tr>
<?php

    $conn_string = "host=localhost port=5432 dbname=cdcol  
                    user=oswald password=geheim";

    $db_handle = pg_connect($conn_string);
    if(!$db_handle)
        die("Kann Datenbank nicht erreichen!");

    $query = "SELECT * FROM cds";

    $result = pg_exec($db_handle, $query);

    if (!$result) 
    {
        echo pg_errormessage($db_handle);
    }
    else
    {
        for ($row = 0; $row < pg_numrows($result); $row++)
        {
            $values = pg_fetch_array($result, $row, PGSQL_ASSOC);
            echo "<tr>";
            echo "<td>".$values['interpret']."</td>";
            echo "<td>".$values['titel']."</td>";
            echo "<td>".$values['jahr']."</td>";
            echo "</tr>";
        }
    }
?>
</table>
</body>
</html>

File: contrib/postgresql.pl
#!/opt/lampp/bin/perl
use DBI;

print "Content-Type: text/html\n\n";

my $dsn="dbi:PgPP:dbname=cdcol;host=localhost;port=5432";
my $dbh=DBI->connect("$dsn","oswald","geheim") or
        die "Kann Datenbank nicht erreichen!";

print "<html>";
print "<head>";
print "<title>Perl und PostgreSQL</title>";
print "</head>";
print "<body>";
print "<h1>Perl und PostgreSQL</h1>";
print "<table border=\"1\">";
print "<tr>";
print "<th>Interpret</th>";
print "<th>Titel</th>";
print "<th>Jahr</th>";
print "</tr>";

my $query="SELECT * FROM cds";

my $prep_sql=$dbh->prepare($query) or  die print "Can't prepare";
$prep_sql->execute() or  die print "Can't  execute";

while (my @row = $prep_sql->fetchrow_array())
{
    print "<tr>";
    print "<td>".$row[2]."</td>";
    print "<td>".$row[1]."</td>";
    print "<td>".$row[3]."</td>";
    print "</tr>";
}

$prep_sql->finish();
$dbh->disconnect();

print "</table>";
print "</body>";
print "</html>";
